app/routes/document-citations.js

```diff
@@ -0,0 +1,15 @@
+const { getResponses } = require('../api/responses')
+
+module.exports = {
+  method: 'GET',
+  path: '/document/{docId}/response/citations',
+  options: {
+    handler: async (request, h) => {
+      const docId = request.params.docId
+      const responses = await getResponses(docId)
+      responses.sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp))
```

Comment:
Not sure if we need this here? All responses should be sorted in desc order anyways from the API.